### DISCUSSION TITLE
Enable/disable dark mode independently from windows theme

### DISCUSSION THREAD

**Discussion by veroddorev** [zoidsh/meru#292] [Q&A] [answered]
Hi, it looks like the color of the inbox depends on the windows theme, without taking into account the background color chosen for gmail. Is there a way to change this ? I'd like to use a lighter theme inside of gmail desktop without having to change the windows one.

**Comment by timche** [comment, marked as answer]
You can set the dark mode at `View` > `Dark Mode`.
